src/Pos/Ssc/GodTossing/GState.hs

```diff
@@ -81,15 +93,66 @@ instance SscGStateClass SscGodTossing where
         view gsOpenings <*>
         view gsShares
 
-loadGlobalState :: (MonadDB m, WithLogger m) => m GtGlobalState
+loadGlobalState
+    :: forall m .
+    ( MonadDB m
+    , WithLogger m
+    , HasContext LrcContext m
+    ) => m GtGlobalState
 loadGlobalState = do
     logDebug "Loading SSC global state"
-    gs <- DB.getGtGlobalState
+    gs <- DB.getGtGlobalState `catch` fallbackLoad
     gs <$ logInfo (sformat ("Loaded GodTossing state: " %build) gs)
+  where
+    fallbackLoad :: SomeException -> m GtGlobalState
```

Comment:
> Also I would rather export getGtGlobalStateMaybe instead of throwing and catching an exception.

It doesn't really matter because it's a workaround.

But catching `SomeException` here is too bad.